Kritika_Sachdeva/LoginModule/EmployeeProject/DbModels/projectContext.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace EmployeeProject.DbModels
+{
+    public partial class projectContext : DbContext
+    {
+        public projectContext()
+        {
+        }
+
+        public projectContext(DbContextOptions<projectContext> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<Temployee> Temployee { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
```

Comment:
Connection string should not be present in Db context file . It should be initialized once in startup class and connection String should be read from app.settings.json

Reply:
made the required changes